VBS/dataset.md

```diff
@@ -0,0 +1,1093 @@
+# List of VBS Analysis Datasets
+
+MC16a
+
+* `4.3G    ./EWVVjj_MGPy8/group.phys-hdbs.mc16_13TeV.361294.CAOD_HIGG5D2.e6456_s3126_r9364_p3717`
+* `3.2G    ./EWVVjj_MGPy8/group.phys-hdbs.mc16_13TeV.361295.CAOD_HIGG5D2.e6456_s3126_r9364_p3717`
+* `213M    ./EWVVjj_MGPy8/group.phys-hdbs.mc16_13TeV.364424.CAOD_HIGG5D2.e5987_s3126_r9364_p3717`
+* `13M     ./EWVVjj_MGPy8/group.phys-hdbs.mc16_13TeV.364425.CAOD_HIGG5D2.e5987_s3126_r9364_p3717`
+* `208M    ./EWVVjj_MGPy8/group.phys-hdbs.mc16_13TeV.364426.CAOD_HIGG5D2.e5987_s3126_r9364_p3717`
+* `14M     ./EWVVjj_MGPy8/group.phys-hdbs.mc16_13TeV.364427.CAOD_HIGG5D2.e5987_s3126_r9364_p3717`
+* `3.7G    ./EWVVjj_MGPy8/group.phys-hdbs.mc16_13TeV.364428.CAOD_HIGG5D2.e5987_s3126_r9364_p3717`
+* `3.4G    ./EWVVjj_MGPy8/group.phys-hdbs.mc16_13TeV.364429.CAOD_HIGG5D2.e5987_s3126_r9364_p3717`
+* `190K    ./MJ_Py8/group.phys-hdbs.mc16_13TeV.361020.CAOD_HIGG5D2.e3569_s3126_r9364_p3639`
+* `326K    ./MJ_Py8/group.phys-hdbs.mc16_13TeV.361021.CAOD_HIGG5D2.e3569_s3126_r9364_p3639`
+* `3.6M    ./MJ_Py8/group.phys-hdbs.mc16_13TeV.361022.CAOD_HIGG5D2.e3668_s3126_r9364_p3639`
+* `26M     ./MJ_Py8/group.phys-hdbs.mc16_13TeV.361023.CAOD_HIGG5D2.e3668_s3126_r9364_p3639`
+* `84M     ./MJ_Py8/group.phys-hdbs.mc16_13TeV.361024.CAOD_HIGG5D2.e3668_s3126_r9364_p3639`
+* `173M    ./MJ_Py8/group.phys-hdbs.mc16_13TeV.361025.CAOD_HIGG5D2.e3668_s3126_r9364_p3639`
+* `304M    ./MJ_Py8/group.phys-hdbs.mc16_13TeV.361026.CAOD_HIGG5D2.e3569_s3126_r9364_p3639`
+* `383M    ./MJ_Py8/group.phys-hdbs.mc16_13TeV.361027.CAOD_HIGG5D2.e3668_s3126_r9364_p3639`
+* `432M    ./MJ_Py8/group.phys-hdbs.mc16_13TeV.361028.CAOD_HIGG5D2.e3569_s3126_r9364_p3639`
+* `424M    ./MJ_Py8/group.phys-hdbs.mc16_13TeV.361029.CAOD_HIGG5D2.e3569_s3126_r9364_p3639`
+* `439M    ./MJ_Py8/group.phys-hdbs.mc16_13TeV.361030.CAOD_HIGG5D2.e3569_s3126_r9364_p3639`
+* `488M    ./MJ_Py8/group.phys-hdbs.mc16_13TeV.361031.CAOD_HIGG5D2.e3569_s3126_r9364_p3639`
+* `460M    ./MJ_Py8/group.phys-hdbs.mc16_13TeV.361032.CAOD_HIGG5D2.e3668_s3126_r9364_p3639`
```

Comment:
@JCZeng1 Things like this appear in here a lot and looking at this

```
group.phys-hdbs.mc16_13TeV.XXXXXX.CAOD_HIGG5D2.e3569_s3126_r9364_p3639
```

it seems these are all from the DSID, but what we want is the DSID that we can put into Rucio.

Reply:
Since I don't produce these samples myself, I don't know the DSID for Rucio.
Is there a way to track them based on what I provided? I couldn't find anything too useful under the EOS directory.

Reply:
@JCZeng1 Groups usually have the DIDs posted on a TWiki someplace for the analysis. In the example I selected here, by looking at the group names and then given that you want DxAODs (your group produces the CxAODs with the CxAOD framework) the following query (if you remove the double reco tags [like I mention](https://gitlab.cern.ch/feickert/jet-cleaning-run-2/-/blob/feat/HCW-2020/run/filelist_MC16.txt)) might be a good start.

```
$ rucio list-dids mc16_13TeV:mc16_13TeV.*.Pythia8EvtGen_*_jetjet_JZ*WithSW.deriv.DAOD_HIGG5D2.e*_s*_r*_p3639 | grep CONTAINER | awk '{print $2}'
mc16_13TeV:mc16_13TeV.364710.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ10WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_r10201_r10210_p3639
mc16_13TeV:mc16_13TeV.364701.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ1WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_s3136_r9364_r9315_p3639
mc16_13TeV:mc16_13TeV.364703.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ3WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364704.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ4WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364700.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ0WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_r9364_r9315_p3639
mc16_13TeV:mc16_13TeV.364703.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ3WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_r10201_r10210_p3639
mc16_13TeV:mc16_13TeV.364710.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ10WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364703.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ3WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_r9364_r9315_p3639
mc16_13TeV:mc16_13TeV.364710.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ10WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_r9364_r9315_p3639
mc16_13TeV:mc16_13TeV.364702.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ2WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_r10201_r10210_p3639
mc16_13TeV:mc16_13TeV.364708.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ8WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_r9364_r9315_p3639
mc16_13TeV:mc16_13TeV.364706.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ6WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_r9364_r9315_p3639
mc16_13TeV:mc16_13TeV.364705.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ5WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_r9364_r9315_p3639
mc16_13TeV:mc16_13TeV.364707.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ7WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364711.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ11WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_r9364_r9315_p3639
mc16_13TeV:mc16_13TeV.364708.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ8WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364712.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ12WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364700.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ0WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_r10201_r10210_p3639
mc16_13TeV:mc16_13TeV.364706.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ6WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364702.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ2WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_s3136_r9364_r9315_p3639
mc16_13TeV:mc16_13TeV.364704.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ4WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_s3136_r10201_r10210_p3639
mc16_13TeV:mc16_13TeV.364712.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ12WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364700.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ0WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_s3136_r9364_r9315_p3639
mc16_13TeV:mc16_13TeV.364709.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ9WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_r10201_r10210_p3639
mc16_13TeV:mc16_13TeV.364705.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ5WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_r10201_r10210_p3639
mc16_13TeV:mc16_13TeV.364701.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ1WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_r9364_r9315_p3639
mc16_13TeV:mc16_13TeV.364705.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ5WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364703.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ3WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_s3136_r10201_r10210_p3639
mc16_13TeV:mc16_13TeV.364712.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ12WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_r9364_r9315_p3639
mc16_13TeV:mc16_13TeV.364700.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ0WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364709.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ9WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_r9364_r9315_p3639
mc16_13TeV:mc16_13TeV.364707.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ7WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_r9364_r9315_p3639
mc16_13TeV:mc16_13TeV.364705.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ5WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_s3136_r9364_r9315_p3639
mc16_13TeV:mc16_13TeV.364703.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ3WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364702.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ2WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364710.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ10WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364702.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ2WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_r9364_r9315_p3639
mc16_13TeV:mc16_13TeV.364704.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ4WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_r10201_r10210_p3639
mc16_13TeV:mc16_13TeV.364704.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ4WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_r9364_r9315_p3639
mc16_13TeV:mc16_13TeV.364711.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ11WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_r10201_r10210_p3639
mc16_13TeV:mc16_13TeV.364711.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ11WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364707.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ7WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_r10201_r10210_p3639
mc16_13TeV:mc16_13TeV.364705.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ5WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_s3136_r10201_r10210_p3639
mc16_13TeV:mc16_13TeV.364701.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ1WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364709.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ9WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364706.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ6WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364708.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ8WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_r10201_r10210_p3639
mc16_13TeV:mc16_13TeV.364707.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ7WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364702.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ2WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364709.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ9WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364701.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ1WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364705.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ5WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364700.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ0WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364711.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ11WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364701.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ1WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_r10201_r10210_p3639
mc16_13TeV:mc16_13TeV.364712.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ12WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_r10201_r10210_p3639
mc16_13TeV:mc16_13TeV.364706.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ6WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_r10201_r10210_p3639
mc16_13TeV:mc16_13TeV.364706.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ6WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_s3136_r9364_r9315_p3639
mc16_13TeV:mc16_13TeV.364708.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ8WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364706.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ6WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_s3136_r10201_r10210_p3639
mc16_13TeV:mc16_13TeV.364702.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ2WithSW.deriv.DAOD_HIGG5D2.e7142_e5984_s3126_s3136_r10201_r10210_p3639
mc16_13TeV:mc16_13TeV.364704.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ4WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
```

Reply:
@JCZeng1 here's that same query with the double reco tags filtered out

```
$ rucio list-dids mc16_13TeV:mc16_13TeV.*.Pythia8EvtGen_*_jetjet_JZ*WithSW.deriv.DAOD_HIGG5D2.e*_s*_r*_p3639 | grep --invert-match "_r.*_r.*_" | grep CONTAINER | awk '{print $2}'
mc16_13TeV:mc16_13TeV.364703.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ3WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364704.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ4WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364710.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ10WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364707.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ7WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364708.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ8WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364712.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ12WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364706.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ6WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364712.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ12WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364705.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ5WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364700.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ0WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364703.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ3WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364702.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ2WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364710.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ10WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364711.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ11WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364701.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ1WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364709.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ9WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364706.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ6WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364707.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ7WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364702.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ2WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364709.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ9WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364701.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ1WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364705.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ5WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364700.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ0WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364711.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ11WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364708.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ8WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364704.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ4WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
```

Reply:
@JCZeng1 Here's a starter script that will even nicely sort them for you (you should of course edit this and make it smarter)

```
(base) [feickert@login ~]$ cat JC_files.sh 
rucio list-dids mc16_13TeV:mc16_13TeV.*.Pythia8EvtGen_*_jetjet_JZ*WithSW.deriv.DAOD_HIGG5D2.e*_s*_r*_p3639 | grep --invert-match "_r.*_r.*_" | grep CONTAINER | awk '{print $2}' &> /tmp/out.txt

printf "# MC16a\n" > sorted.txt
grep r9364 /tmp/out.txt | sort >> sorted.txt 
printf "\n# MC16d\n" >> sorted.txt
grep r10201 /tmp/out.txt | sort >> sorted.txt 
(base) [feickert@login ~]$ . JC_files.sh 
(base) [feickert@login ~]$ cat sorted.txt 
# MC16a
mc16_13TeV:mc16_13TeV.364700.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ0WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364701.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ1WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364702.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ2WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364703.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ3WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364704.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ4WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364705.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ5WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364706.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ6WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364707.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ7WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364708.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ8WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364709.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ9WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364710.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ10WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364711.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ11WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639
mc16_13TeV:mc16_13TeV.364712.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ12WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r9364_p3639

# MC16d
mc16_13TeV:mc16_13TeV.364700.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ0WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364701.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ1WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364702.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ2WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364703.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ3WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364704.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ4WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364705.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ5WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364706.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ6WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364707.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ7WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364708.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ8WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364709.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ9WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364710.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ10WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364711.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ11WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
mc16_13TeV:mc16_13TeV.364712.Pythia8EvtGen_A14NNPDF23LO_jetjet_JZ12WithSW.deriv.DAOD_HIGG5D2.e7142_s3126_r10201_p3639
```

Reply:
@JCZeng1 I realize now though that you had listed `e3569_s3126_r9364_p3639` and none of the above of my output match the event gen tag of `e3569` that you had. So you should probably ask people in your group if there is a full list of DxAODs.